### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by pillip8282**

### CODE REVIEW

File: external/wakaama/core/er-coap-13/er-coap-13.c [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 #include "er-coap-13.h"
 
-#include "liblwm2m.h"			/* for lwm2m_malloc() and lwm2m_free() */
+#include <protocols/liblwm2m.h>			/* for lwm2m_malloc() and lwm2m_free() */
```
Comment (sunghan-chang) [inline external/wakaama/core/er-coap-13/er-coap-13.c]: The Wakaama is in external folder. Do you want to modify this in all of wakaama files?
How about add including path of "external/include/protocols" in makefile?
That is easier to maintain I think.

File: external/include/protocols/liblwm2m.h [hunk 2]
```diff
@@ -71,44 +54,49 @@
...
 extern "C" {
-#else
-#define EXTERN extern
 #endif
 
 #include <stdint.h>
 #include <stddef.h>
 #include <stdbool.h>
-#include <sys/time.h>
+#include <time.h>
```
Comment (sunghan-chang) [inline external/include/protocols/liblwm2m.h]: I don't know why you modify this file. Is it updating of Wakaama from open source?

File: os/Kconfig [hunk 3]
```diff
@@ -501,3 +501,11 @@ endmenu
 menu "Application Configuration"
 source "$APPSDIR/Kconfig"
 endmenu
+
+menu "Runtime Environment"
+source Kconfig.runtime
```
Comment (sunghan-chang) [inline os/Kconfig]: Where are they, runtime and devmng?

Comment (pillip8282) [reply os/Kconfig]: I forgot to upload those.

File: external/wakaama/core/er-coap-13/er-coap-13.c (hunk 1 above)
Comment (pillip8282) [reply external/wakaama/core/er-coap-13/er-coap-13.c]: unfortunately wakaama also refers er-coap in "wakaama/core"  if I didn't change an include path. I should remove or sync the file liblwm2m.h in "wakaama/core". Actually I don't know which is better, but I think chaning path is clear than other methods.

File: external/include/protocols/liblwm2m.h (hunk 2 above)
Comment (pillip8282) [reply external/include/protocols/liblwm2m.h]: I think it's the replace issue. I didn't change it. you can check <time.h> in original path "external/wakaama/core/liblwm2m.h". actually now we are trying to merge those two files in conflict. we will consider to the issue when we are merging it.

File: external/wakaama/examples/shared/commandline.c [hunk 4]
```diff
@@ -22,7 +22,7 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <inttypes.h>
-#include "liblwm2m.h"
+#include <protocols/libl2mwm.h>
```
Comment (dr-venkman) [inline external/wakaama/examples/shared/commandline.c]: Please check headerfilename: liblwm2m.h is the right filename.

File: external/wakaama/examples/shared/dtlsconnection.h [hunk 5]
```diff
@@ -31,7 +31,7 @@
 
 #include "tinydtls/tinydtls.h"
 #include "tinydtls/dtls.h"
-#include "liblwm2m.h"
+#include <protocols/libl2mwm.h>
```
Comment (dr-venkman) [inline external/wakaama/examples/shared/dtlsconnection.h]: please check filename as above.

File: framework/include/dm/dm_lwm2m.h [hunk 6]
```diff
@@ -34,7 +34,7 @@
 #include <tinyara/config.h>
 
 #ifdef CONFIG_LWM2M_WAKAAMA
-#include "liblwm2m.h"
+#include <protocols/libl2mwm.h>
```
Comment (dr-venkman) [inline framework/include/dm/dm_lwm2m.h]: please check filename as above.

File: os/Kconfig.devmng [hunk 7]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (sunghan-chang) [inline os/Kconfig.devmng]: 1. need to add belows.
```
#
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```
2. DM is in framework. Let's move this inside framework/src/dm.

File: os/Kconfig [hunk 8]
```diff
@@ -501,3 +501,11 @@ endmenu
...
+endmenu
+
+menu "Device Management"
```
Comment (sunghan-chang) [inline os/Kconfig]: Do you need to pick this out from framework?

File: external/wakaama/examples/bootstrap_server/bootstrap_info.h [hunk 9]
```diff
@@ -16,7 +16,7 @@
  *******************************************************************************/
 
 #include <stdio.h>
-#include <protocols/libl2mwm.h>
+#include <protocols/liblwm2m.h>
```
Comment (sunghan-chang) [inline external/wakaama/examples/bootstrap_server/bootstrap_info.h]: This commit is "os:Kconfig Add new menuconfigs(Runtime, Device Manage)". You can't add these changes in here.

File: os/Kconfig.devmng (hunk 7 above)
Comment (pillip8282) [reply os/Kconfig.devmng]: ok

File: os/Kconfig (hunk 8 above)
Comment (pillip8282) [reply os/Kconfig]: yes, we agreed it on the document

**Comment by pillip8282** [comment]
I splited the pull request 600, 601 becuase those commits are not tightly related.

**Pull request closed without merging**